airbyte-integrations/connectors/source-typeform/unit_tests/test_source.py

```diff
@@ -0,0 +1,57 @@
+from airbyte_cdk import AirbyteLogger
+
+from source_typeform import SourceTypeform
+from source_typeform.source import TypeformStream
+
+logger = AirbyteLogger()
+
+TYPEFORM_BASE_URL = TypeformStream.url_base
+
+
+def test_check_connection(requests_mock, config, empty_response_ok):
+    requests_mock.register_uri("GET", TYPEFORM_BASE_URL + "me", empty_response_ok)
+    requests_mock.register_uri("GET", TYPEFORM_BASE_URL + "forms/u6nXL7", empty_response_ok)
+    requests_mock.register_uri("GET", TYPEFORM_BASE_URL + "forms/k9xNV4", empty_response_ok)
+
+    ok, error = SourceTypeform().check_connection(logger, config)
+
+    assert ok and not error
+
+
+def test_check_connection_bad_request_me(requests_mock, config, empty_response_bad):
+    requests_mock.register_uri("GET", TYPEFORM_BASE_URL + "me", empty_response_bad)
+
+    ok, error = SourceTypeform().check_connection(logger, config)
+
+    assert not ok and error
```

Comment:
Can you capture the correct output/error and check it? Maybe use pytest match feature

Reply:
matched using re.match()